src/bot.js

```diff
@@ -12,21 +14,22 @@ const bot = new Discord.Client({
 });
 
 // Import commands
-const commands = require('./commands/index');
+const commands = require('./commands');
 const commandList = Object.keys(commands);
 
 bot.once('ready', async () => {
   const guild = bot.guilds.cache.get(config.guildId);
+
   // Remove all commands (To flush out unused commands)
   const collectedCommands = await guild.commands.fetch();
-  collectedCommands.forEach(async (command) => {
+  for (const command of collectedCommands) {
     await guild.commands.delete(command);
-  });
```

Comment:
A better way to handle asynchronous looping is using `for...of` syntax (details [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)).